drivers/clock_control/nrf_power_clock.c

```diff
@@ -10,8 +10,8 @@
 #include <drivers/clock_control.h>
 #include <drivers/clock_control/nrf_clock_control.h>
 #include "nrf_clock_calibration.h"
+#include <nrfx_clock.h>
 #include <logging/log.h>
-#include <hal/nrf_power.h>
```

Comment:
Without `#include <nrfx_power.h>` samples for 5340 will not compile as `nrfx_power.h` includes `"nrfx_power_compat.h"` with usbreg stuff (for 5340).

Reply:
moved all usb related stuff to `usb_dc_nrfx.c`